webpack.config.js

```diff
@@ -6,11 +6,12 @@ const HOST = process.env.HOST || '127.0.0.1';
 const PORT = process.env.PORT || '8080';
 
 module.exports = {
+  mode: 'production',
   target: 'web',
   entry: {
     index: './example/client.jsx',
   },
-  devtool: 'cheap-module-eval-source-map',
+  devtool: 'eval-cheap-module-source-map',
```

Comment:
For testing against webpack 5 to work, this change is required, and the other 2 fix warnings.